src/client/libcurve_file.cpp

```diff
@@ -153,8 +154,12 @@ int FileClient::Init(const std::string& configpath) {
         return -LIBCURVE_ERROR::FAILED;
     }
 
-    auto tmpMdsClient = std::make_shared<MDSClient>();
+    // init auth client
+    AuthClient::GetInstance().Init(
+        clientconfig_.GetFileServiceOption().metaServerOpt,
+        clientconfig_.GetFileServiceOption().authClientOption);
```

Comment:
After this patch https://github.com/opencurve/curve/commit/8db694299e7952974233c1a738d3099cb7b448d1, client supports mount volumes from different clusters, but `AuthClient` is a singleton, so we may have problems in this case.

Reply:
fixed